plugins/labhub.py

```diff
@@ -381,3 +381,100 @@ def pr_stats(self, msg, match):
                     state=type(self).community_state(pr_count)
                  )
         yield reply
+
+    @re_botcmd(pattern=r'^migrate\s+https://github.com/([^/]+)/([^/]+)/+issues/(\d+)\s+https://github.com/([^/]+)/([^/]+)/*$',  # Ignore LineLengthBear, PyCodeStyleBear
+               # Ignore LineLengthBear, PyCodeStyleBear
+               re_cmd_name_help='migrate <complete-issue-URL-to-be-copied-from> <complete-repo-URL-to-be-copied-into>',
+               flags=re.IGNORECASE)
+    def migrate_issue(self, msg, match):
+        """
+        Migrate an issue from one repo
+        to another repo of coala
+        """
+        org = match.group(1)
+        repo_name_orig = match.group(2)
+        issue_number = match.group(3)
+        user = msg.frm.nick
+        org2 = match.group(4)
+        repo_name_final = match.group(5)
+
+        try:
+            assert org == self.GH_ORG_NAME or org == self.GL_ORG_NAME
+        except AssertionError:
+            yield 'First repository not owned by our org.'
+            return
+
+        try:
+            assert org2 == self.GH_ORG_NAME or org2 == self.GL_ORG_NAME
+        except AssertionError:
+            yield 'Second repository not owned by our org.'
+            return
+
+        if (repo_name_orig in self.REPOS) and (repo_name_final in self.REPOS):
```

Comment:
use, `if not...` so we can remove the reduntant if block and the contained `pass` statement